http/src/main/resources/reference.conf

```diff
@@ -285,6 +285,52 @@ azure {
      ]
    }
 
+  workflows-app-config {
+    instrumentation-enabled = false
+    chart-name = "/leonardo/workflows-app"
+    chart-version = "0.1.0"
+    release-name-suffix = "workflows-app-rls"
+    namespace-name-suffix = "workflows-app-ns"
+    ksa-name = "workflows-app-ksa"
+    dockstore-base-url = "https://staging.dockstore.org/"
+    services = [
+      {
+        name = "cbas"
+        kind = "ClusterIP"
+      },
+      {
+        name = "cromwell"
+        kind = "ClusterIP"
+      }
+    ]
+    # Not enabled to make sure people cannot start the (not yet complete) app:
+    enabled = false
+    database-enabled = false
+    chart-versions-to-exclude-from-updates = []
+  }
+
+  cromwell-runner-app-config {
+    instrumentation-enabled = false
+    chart-name = "/leonardo/cromwell-runner-app"
+    chart-version = "0.1.0"
+    release-name-suffix = "cromwell-runner-app-rls"
+    namespace-name-suffix = "cromwell-runner-app-ns"
+    ksa-name = "cromwell-runner-app-ksa"
+    dockstore-base-url = "https://staging.dockstore.org/"
+    # See https://github.com/broadinstitute/cromwhelm/blob/main/terra-batch-libchart/templates/_reverse-proxy.tpl#L81-L114
+    # for list of services.
+    services = [
+      {
+        name = "cromwell"
```

Comment:
@rtitle since Chris is on vacation, I am working on getting this PR merged. I am not sure if names for the services need to be unique or not. @Qi77Qi or @LizBaldo do you know if `/cromwell` path for 2 different apps will clash or not?

Reply:
I do not know if it will crash, but for better clarity it might be nice to name them differently regardless?

Reply:
don't think so, but it depends on your helmchart, I don't know if it expects to use a specific service name

Reply:
Hmm I am not so sure. Since this PR is meant to get us started on the new app types (nothing is deployed yet), I will rename the `/cromwell` paths and if things need to be updated in helm charts they can be done as part of another PR